tap_shopify/schemas/inventory_items.json

```diff
@@ -4,7 +4,7 @@
     "id": {
       "type": [
         "null",
-        "integer"
```

Comment:
shopify gql only supports id's in following pattern: `gid://shopify/InventoryItem/435374857348"`